dev-tools/src/lambdas/dynamoOperations/handler.ts

```diff
@@ -14,11 +14,6 @@ export const handler = async (
   context: Context
 ) => {
   initialiseLogger(context)
-  logger.info(
```

Comment:
For these logs that show the function parameter... is this not useful information? In my mind this is how you would identify which group of logs is relevant to your invocation. E.g. zendesk id, etc.

This is more useful than logging in the switch statement if you ask me!

Reply:
Yep makes sense! Although left a comment around wording :)